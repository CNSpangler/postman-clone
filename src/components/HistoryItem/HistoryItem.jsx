import React from 'react';
import PropTypes from 'prop-types';
import ReactJson from 'react-json-view';

const HistoryItem = ({ item }) => {
  return (
    <>
      <ReactJson src={item} />
      <h1>HISTORY ITEM</h1>
    </>
  );
};

HistoryItem.propTypes = {
  item: PropTypes.object,
  url: PropTypes.string,
  method: PropTypes.string,
  body: PropTypes.any
};

export default HistoryItem;
