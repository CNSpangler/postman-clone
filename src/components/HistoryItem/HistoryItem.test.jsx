import React from 'react';
import { shallow } from 'enzyme';
import HistoryItem from './HistoryItem.jsx';

describe('HistoryItem component', () => {
  it('renders HistoryItem ', () => {
    const wrapper = shallow(<HistoryItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
