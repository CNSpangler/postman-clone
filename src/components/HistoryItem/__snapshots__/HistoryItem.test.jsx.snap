// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryItem component renders HistoryItem  1`] = `
<Fragment>
  <t
    collapseStringsAfterLength={false}
    collapsed={false}
    defaultValue={null}
    displayDataTypes={true}
    displayObjectSize={true}
    enableClipboard={true}
    groupArraysAfterLength={100}
    iconStyle="triangle"
    indentWidth={4}
    name="root"
    onAdd={false}
    onDelete={false}
    onEdit={false}
    onSelect={false}
    shouldCollapse={false}
    sortKeys={false}
    src={Object {}}
    style={Object {}}
    theme="rjv-default"
    validationMessage="Validation Error"
  />
  <h1>
    HISTORY ITEM
  </h1>
</Fragment>
`;
